src/stats.c

```diff
@@ -1127,10 +1130,14 @@ void ClearError ( void )
         }
 #ifdef USE_GASMAN
         /* and check if maximal memory was overrun */
-        if ( SyStorOverrun != 0 ) {
-          SyStorOverrun = 0; /* reset */
-          Pr("GAP has exceeded the permitted memory (-o option),\n", 0, 0);
-          Pr("the maximum is now enlarged to %d kB.\n", (Int)SyStorMax, 0);
+        if (SyStorOverrun != SY_STOR_OVERRUN_CLEAR) {
```

Comment:
Can't you just remove this line?
```suggestion
```
Well, and then also the matching `}`, and the indentation needs to be adjusted.